web/src/inspector.js

```diff
@@ -528,6 +529,23 @@ export class Inspector extends EventTarget {
       string = torrents.every((t) => get(t) === first) ? first : mixed;
     }
     setTextContent(e.info.location, string);
+
+    // magnetLink
+    if (torrents.length === 0) {
+      string = none;
+    } else {
+      const get = (t) => t.getMagnetLink();
+      const first = get(torrents[0]);
+      string = torrents.every((t) => get(t) === first) ? first : mixed;
+    }
+    if (none !== string) {
+      Utils.setInnerHTML(
+        e.info.magnetLink,
+        `<a class="inspector-info-magnet" href="${string}"><button></button></a>`
+      );
+    } else {
+      setTextContent(e.info.magnetLink, string);
+    }
```

Comment:
Since magnet links include an escaped version of the torrent's info-hash, we know that `torrents.every()` will always fail unless length ==== 1

```suggestion
    // magnetLink
    if (torrents.length === 0) {
      setTextContent(e.info.magnetLink, none);
    } else if (torrents.length > 1) {
      setTextContent(e.info.magnetLink, mixed);
    } else {
      const link = torrents[0].getMagnetLink();
      Utils.setInnerHTML(
        e.info.magnetLink,
        `<a class="inspector-info-magnet" href="${link}"><button></button></a>`
      );
  }
```